### PR TITLE
Update guide page for audio management - TCSACR-175

### PR DESCRIPTION
**Pull request by hsgwon**
### Change Description ###

Update guide page for audio management

### API Changes ###

- TCSACR-175

### CODE REVIEW

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/501

File: docs/application/dotnet/guides/multimedia/audio.md [hunk 1]
```diff
@@ -133,12 +133,12 @@ To query audio device information:
...
         }
 
-        AudioManager.DeviceStateChanged += OnDeviceStateChanged;
+        AudioManager.DeviceRunningChanged += OnDeviceRunningChanged;
         ```
 
 
         > **Note**   
-		> The initial state of the connected device is `Deactivated`.
+		> The initial running state of the connected device is `false', that is 'not running'.
```
Comment (journey2w) [inline docs/application/dotnet/guides/multimedia/audio.md]: There is an tag error.  The code tag is \`  \`.

How about the following sentence?
```
> The initial running state of the connected device is `false`, which means the connected device is not running.
```

Comment (hsgwon) [reply docs/application/dotnet/guides/multimedia/audio.md]: Fixed

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/501

**Comment by mijin80lee** [comment]
TD Sync Completed.

**Pull request merged**